Sources/MongoSwift/ReadPreference.swift

```diff
@@ -96,6 +96,11 @@ public final class ReadPreference {
         self._readPreference = mongoc_read_prefs_new(mode.readMode)
     }
 
+    /// Initializes a new ReadPreference with the default mode (primary).
+    public init() {
```

Comment:
because this is a class and this initializer now delegates to a different one, you need to add the `convenience` keyword. [This page](https://docs.swift.org/swift-book/LanguageGuide/Initialization.html) gives a good overview of the rules around `init`.

Reply:
there are a few places in the driver we explicitly create a `.primary` read preference to serve as a default. I think it would be good to update those places to call this new init